lib/src/main/java/org/openbase/bco/registry/lib/com/DefaultMessageFilter.java

```diff
@@ -0,0 +1,75 @@
+package org.openbase.bco.registry.lib.com;
```

Comment:
move to jul

Reply:
Renamed to MockUpFilter which works for any type, not just messages, and moved to jul